### PR TITLE
Jasmine awesome

### PR DESCRIPTION
**Pull request by jalvar53**
Adding jasmine-awesome-reporter the tests

### CODE REVIEW

File: protractor/helpers/awesome-report.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+export let awesomeReporter = () => {
+  const config = {
+    fullPath: 'reports',
+    fileName: 'awesome',
+    merge: true
+  };
+  jasmine.getEnv().addReporter(AwesomeReport.getReport(config));
+};
+
+
```
Comment (germandavid85) [inline protractor/helpers/awesome-report.ts]: remove extra line

**Review by germandavid85 — CHANGES_REQUESTED** [review]

File: src/page/ProductDetail.page.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { $, ElementFinder, promise } from 'protractor';
 
 export class ProductDetailPage {
   private get cartMenu(): ElementFinder {
-    return $('#add_to_cart > button > span');
+    return $('[name="Submit"] > span');
```
Comment (luisbolanoss) [inline src/page/ProductDetail.page.ts]: review that this selector to be unique

File: protractor/headless.config.ts [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import { browser, Config } from 'protractor';
 import { reporter } from './helpers/reporter';
+import { awesomeReporter } from './helpers/awesome-report';
 
 export const config: Config = {
   framework: 'jasmine',
   jasmineNodeOpts: {
-    defaultTimeoutInterval: 120000
+    defaultTimeoutInterval: 120000,
```
Comment (germandavid85) [inline protractor/headless.config.ts]: remove this comman

File: src/page/ProductDetail.page.ts (hunk 2 above)
Comment (germandavid85) [inline src/page/ProductDetail.page.ts]: this change is not part of the commit changes, it should go in a separate commit

**Review by germandavid85 — CHANGES_REQUESTED** [review]

**Review by germandavid85 — APPROVED** [review]

**Pull request merged**